deprecated.csv

```diff
@@ -1,16 +1,18 @@
-libpq, 9.6.6, 2023-07-06, "libpq is now built as part of postgresql-feedstock"
-unittest2, 1.1.0, 2023-07-06, "Deprecated. Python 2 library"
-theano, 1.0.5, 2023-07-06, "Deprecated library."
-odo, 0.5.1, 2023-07-06, "Deprecated library."
-backports.lzma, 0.0.14, 2023-07-06, "Deprecated library for Python versions before 3.3"
-backports.functools_lru_cache, 1.6.4, 2023-07-06, "Deprecated library for older Python versions"
-backports.os, 0.1.1, 2023-07-06, "Deprecated library for older Python versions"
-backports.shutil_get_terminal_size, 1.0.0, 2023-07-06, "Deprecated library for older Python versions"
-backports.shutil_which, 3.5.2, 2023-07-06, "Deprecated library for older Python versions"
-backports.tempfile, 1.0, 2023-07-06, "Deprecated library for older Python versions"
-backports.weakref, 1.0.post1, 2023-07-06, "Deprecated library for older Python versions"
-backports_abc, 0.5, 2023-07-06, "Deprecated library for older Python versions"
-airflow, 2.4.3, 2023-08-30, "Deprecated due to small d/l count. There are official Docker images that are supported."
\ No newline at end of file
+package, feedstock, last_version, date, reason
+salt, salt-feedstock, 2018.3.3, 2023-06-29, "salt is an automation and management framework and is not aligned with current goals."
+gymnasium-notices, gymnasium-notices-feedstock, 0.0.1, 2023-07-06, "Replaced by farama-notifications. Upstream renaming."
+,libpq-feedstock, 9.6.6, 2023-07-06, "libpq is now built as part of postgresql-feedstock"
+unittest2,unittest2-feedstock, 1.1.0, 2023-07-06, "Deprecated. Python 2 library"
+theano,theano-feedstock, 1.0.5, 2023-07-06, "Deprecated library."
+odo,odo-feedstock, 0.5.1, 2023-07-06, "Deprecated library."
+backports.lzma, backports.lzma-feedstock, 0.0.14, 2023-07-06, "Deprecated library for Python versions before 3.3"
+backports.functools_lru_cache ,backports.functools_lru_cache-feedstock, 1.6.4, 2023-07-06, "Deprecated library for older Python versions"
+backports.os, backports.os-feedstock, 0.1.1, 2023-07-06, "Deprecated library for older Python versions"
+backports.shutil_get_terminal_size, backports.shutil_get_terminal_size-feedstock, 1.0.0, 2023-07-06, "Deprecated library for older Python versions"
+backports.shutil_which, backports.shutil_which-feedstock, 3.5.2, 2023-07-06, "Deprecated library for older Python versions"
+backports.tempfile, backports.tempfile-feedstock, 1.0, 2023-07-06, "Deprecated library for older Python versions"
+backports.weakref, backports.weakref-feedstock, 1.0.post1, 2023-07-06, "Deprecated library for older Python versions"
+backports_abc, backports_abc-feedstock, 0.5, 2023-07-06, "Deprecated library for older Python versions"
+airflow, airflow-feedstock, 2.4.3, 2023-08-30, "Deprecated due to small d/l count. There are official Docker images that are supported."
+, pytorch-cpu, 1.3.1, 2023-09-27, "This was an old feedstock that is no longer in use.  It was moved to [old_root_feedstocks](old_root_feedstocks/pytorch_cpu) for archival."
```

Comment:
For libpq, I see that we added a note on where the package is now being built (postgresql-feedstock). I feel like we should probably do the same here.